ql/src/java/org/apache/hadoop/hive/ql/optimizer/calcite/HiveTezModelRelMetadataProvider.java

```diff
@@ -58,4 +60,24 @@
                 HiveRelMdPredicates.SOURCE,
                 JaninoRelMetadataProvider.DEFAULT)));
 
+  public static JaninoRelMetadataProvider with(RelOptMaterialization materialization) {
+    return JaninoRelMetadataProvider.of(
+            ChainedRelMetadataProvider.of(
+                    ImmutableList.of(
+                            HiveRelMdDistinctRowCount.SOURCE,
```

Comment:
can't we reuse somehow the other list(`DEFAULT`):

I think something like this could probably work:
``` 
  public static final JaninoRelMetadataProvider DEFAULT2 =
      JaninoRelMetadataProvider.of(
        ChainedRelMetadataProvider.of(
            ImmutableList.of(
                HiveRelMdRowCount.SOURCE,
                DEFAULT)));
```

this provider could probably also be moved closer to the place where its being used

Reply:
Done.